CONTRIBUTING.md

```diff
@@ -1,165 +1,77 @@
-
-## Description
-(high-level summary that will be accessible to product/UX/Test, plus link to Storybook, explain the different cases)
-
-## When to use this component
-(including where it is currently used)
-
-## When not to use this component
-(if this is not known, can leave the section empty)
-
-## Installation
-`npm install (package-name)`
-
-## Props
-(a table of all the component's props, following the template below)
 
-| Argument  | Type                | Required | Default | Example         |
-|-----------|---------------------|----------|---------|-----------------|
-| prop-name | description of type | Yes/No   | `value` | `example input` |
+### Edit existing packages
 
-## Usage
-(A code block containing an example usage of the component, including importing)
+1. Find a Psammead component you wish you to work on, in your project from the [directory of available packages](https://github.com/bbc/psammead/tree/latest/packages), or by running `npx lerna list` inside the repo.
+2. To install the package in your local development, run:
+   ```
+   npm install @bbc/psammead-<component_name>
+   ```
+3. See the README for the respective the Psammead component, to see the required props, usage example and use-cases for it.
 
-## Accessibility notes
-(ensuring they make sense to non-technical audiences)
+### Using multiple components locally
 
-## Roadmap
-(what we have planned for this component)
+When making changes to a package locally if you want to pull those changes into another psammead package then the following command will create the required symlinks for you.
 
-## Additional notes
-(link to any relevant ADRs)
+Run the following command to link all psammead packages up regardless of dependency version:
 
-## Contributing
-
-Psammead is completely open source. We are grateful for any contributions, whether they be new components, bug fixes or general improvements. Please see our primary contributing guide which can be found at [the root of the Psammead respository](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md).
-
-### [Code of Conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)
-
-We welcome feedback and help on this work. By participating in this project, you agree to abide by the [code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md). Please take a moment to read it.
-
-### License
-
-Psammead is [Apache 2.0 licensed](https://github.com/bbc/psammead/blob/latest/LICENSE).
 ```
-
-When linking to other pages from a component's README, we advise you do so only by absolute URL, and avoid the use of relative paths. Component READMEs are displayed in a variety of places, including NPM and Storybook, and this helps ensure the intended meaning is preserved.
-
-If you wish to exclude one or more sections of this template, we advise you do so by simply commenting out the heading, like so:
-
-```
-<!-- ## Roadmap -->
+npm run install:packages --force-local
```

Comment:
I'm not sure, but doesn't this need an extra `--`? 

Like `npm run install:packages -- --force-local`

Reply:
I don't think so @lokh01 ?

Reply:
Doesn't need it. 
![image](https://user-images.githubusercontent.com/53564281/65222794-bb6f1380-dab7-11e9-9f2e-df9f3c558728.png)